vision/pylon/detect_pylon.py

```diff
@@ -0,0 +1,28 @@
+import cv2
+import numpy as np
+
+# detect_red detects all pixels that fall in a certain range in an image
+# then outputs the original image and detected mask
+def detect_red(image):
+    # Convert the image from BGR to HSV
+    hsv = cv2.cvtColor(image, cv2.COLOR_BGR2HSV)
+
+    # Set the lower and upper color limits
+    lower_red = np.array([50, 150, 25])
+    upper_red = np.array([255, 255, 120])
```

Comment:
We should make these into constants -- ie variable names to all caps, and define them outside of the function definition. This way they can be easily changed by anyone.